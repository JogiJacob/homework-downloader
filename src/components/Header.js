import React from 'react'

function Header() {
    return (
        <div className="Title">
            <h3>Downloader</h3>
        </div>
    )
}

export default Header
