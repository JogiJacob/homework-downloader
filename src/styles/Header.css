.Title {
    box-sizing: border-box;
    width: 100%;
    display: flex;
    justify-content: center;
    align-items: baseline;
    padding: 0.3em;
    margin-bottom: 2em;
    background-color: #1a9ac0;
    color: #fff;
  }