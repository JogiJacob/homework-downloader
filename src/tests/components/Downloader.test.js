import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Downloader';

test('should test Downloader component', () => {
 const wrapper = shallow(<Header />);
 expect(wrapper).toMatchSnapshot();
});