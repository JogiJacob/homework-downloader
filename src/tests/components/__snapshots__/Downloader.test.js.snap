// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Downloader component 1`] = `
<Fragment>
  <TopBar
    selectedItems={Array []}
    selectionText="None Selected"
  />
  <TableComponent
    data={Array []}
    setSelections={[Function]}
  />
</Fragment>
`;
